modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PythonClientCodegen.java

```diff
@@ -694,6 +694,158 @@ public String toRegularExpression(String pattern) {
         return addRegularExpressionDelimiter(pattern);
     }
 
+    @Override
+    public String toExampleValue(Schema schema) {
+        return toExampleValueRecursive(schema, new ArrayList<String>());
+    }
+
+    private String toExampleValueRecursive(Schema schema, List<String> included_schemas) {
```

Comment:
How about passing in an indentation level?
That way when defining a nested list or dict or child model, they will be cleanly inline indented and will minimize the chances that we will need an line too long E501 opt out for flake8 linting.

Reply:
Same problem than above. I would have to let java know if it is include_optional's code or not, since include_optional part is more indented to start with.

Reply:
What if we switch back to indenting the else clause like you initially had. Then the indentation in either location starts at the same level. Also if you made make instance a function then you could remove an indentation level. What do you think?

Reply:
Yes, it is a proper approach

Reply:
Done. Will not work on the project for around 2 weeks, please consider to accept the PR.